types/src/transaction/transaction_v1/transaction_v1_body/arg_handling.rs

```diff
@@ -6,13 +6,12 @@ use super::super::TransactionV1ConfigFailure;
 use crate::{
     account::AccountHash,
     bytesrepr::{FromBytes, ToBytes},
-    CLTyped, CLValue, CLValueError, PublicKey, RuntimeArgs, URef, U512,
+    CLType, CLTyped, CLValue, CLValueError, PublicKey, RuntimeArgs, TransferTarget, URef, U512,
 };
 
-const TRANSFER_ARG_SOURCE: RequiredArg<URef> = RequiredArg::new("source");
-const TRANSFER_ARG_TARGET: RequiredArg<URef> = RequiredArg::new("target");
 const TRANSFER_ARG_AMOUNT: RequiredArg<U512> = RequiredArg::new("amount");
-const TRANSFER_ARG_TO: OptionalArg<AccountHash> = OptionalArg::new("to");
+const TRANSFER_ARG_SOURCE: OptionalArg<URef> = OptionalArg::new("source");
+const TRANSFER_ARG_TARGET: &str = "target";
```

Comment:
Yes, but it unfortunately doesn't fit into my `RequiredArg` type as it can actually be any one of three types: `PublicKey`, `AccountHash` or `URef`.  I had to special-case for it in `fn new_transfer_args`.